drivers/sensor/lsm6dsl/Kconfig

```diff
@@ -8,11 +8,11 @@
 #
 
 menuconfig LSM6DSL
-	bool "LSM6DSL I2C accelerometer and gyroscope Chip"
-	depends on I2C
+	bool "LSM6DSL I2C/SPI accelerometer and gyroscope Chip"
+	depends on SENSOR && (I2C || SPI)
 	default n
 	help
-	  Enable driver for LSM6DSL I2C-based accelerometer and gyroscope
+	  Enable driver for LSM6DSL accelerometer and gyroscope
 	  sensor.
 
 if !HAS_DTS_I2C_DEVICE
```

Comment:
I think you should move this below "choice/endchoice" section and put it around I2C config for now

Reply:
ooops... yes correct!